tools/build_post_process.py

```diff
@@ -407,10 +426,9 @@ def cleanup_files(deps_dir, libs_dir):
 
     find_progress_bar.close()
 
-    to_delete.append(libs_dir / "ayon")
-    to_delete.append(libs_dir / "ayon.pth")
-    to_delete.append(deps_dir / "ayon.pth")
-
+    to_delete.extend(
+        (libs_dir / "ayon", libs_dir / "ayon.pth", deps_dir / "ayon.pth")
+    )
```

Comment:
Admittedly, the old way made it much easier to see the subtle differences between each entry.